tools/rum-models-generator/Sources/RUMModelsGeneratorCore/Output/Printing/ObjcInteropPrinter.swift

```diff
@@ -241,7 +241,7 @@ internal class ObjcInteropPrinter: BasePrinter {
         let objcPropertyName = swiftProperty.name
         let objcEnumName = objcTypeNamesPrefix + nestedObjcEnum.objcTypeName
 
-        if swiftProperty.isMutable {
+        if swiftProperty.mutability == .mutable {
```

Comment:
How about using `switch` instead of `if` here and in other places in this class? Doing that is a bit more repetitive, but has the benefit of compiler explicitly guiding us when adding new cases to `Mutability` enum in the future:
```swift
switch swiftProperty.mutability {
   case .mutable:
      // the `if {}` code
   case .immutable, .mutableInternally:
      // the `else {}` code
}
```